File: java/src/main/java/array/Array.java
package array;

public class Array {
    public static void printArray(int arr[], int size) {
        int i;
        for (i = 0; i < size; i++)
            System.out.print(arr[i] + " ");
        System.out.println();
    }

    public static Boolean sorted(int arr[], int size) {
        for (int i = 1; i < size; i++) {
            if (!(arr[i-1] < arr[i])) {
                return false;
            }
        }
        return true;
    }
}

File: go/array/array.go
package array

import "fmt"

func IsSorted(arr []int, size int) bool {
    for i := 1; i < size; i++ {
        if arr[i-1] > arr[1]{
            return false;
        }
    }
    return true;
}

func ToString(arr []int, size int) string {
    str := "";

    str += fmt.Sprintf("[%d",arr[0]);

    for i := 1; i < size; i++ {
        str += fmt.Sprintf(", %d", arr[i])
    }
    
    str += "]";
    return str;
}
